app/components/school_information/details/head_component.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-6">
+      <%= @school.name %>
+    </h1>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-m"><%= @selected_cohort.start_year %> recruitment</h2>
+  </div>
+</div>
```

Comment:
These are both a row with two-thirds width. Can we make it one?

Reply:
I tried - looked ugly - styles from Al input. Going to keep them as they are